src/components/CvViewer/CvViewer.types.ts

```diff
@@ -8,38 +8,46 @@ export enum Template {
   ModernColumnBlack = 'modern column black'
 }
 
+export type PersonalInfoType = {
+  jobTitle?: string;
+  fullName?: string;
+  email?: string;
+  address?: string;
+  avatar?: string;
+  isShowAvatar?: boolean;
+  description?: string;
+};
+
+export type WorkExperienceType = {
+  id: string;
+  companyName?: string;
+  position?: string;
+  dateTime?: string[]; // american date format
+  startDate?: string; // american date format
+  endDate?: string; // american date format
+  isCurrentWork?: boolean;
+  description?: string;
+  remark?: string;
```

Comment:
Maybe, here it need to be rewritten to description instead of remark

Reply:
As I've just mentioned above probably remain remark is ok. For me it looks like (remark - is some addition to your experience whether it is job or education experience, when description - an AI answer), so we need to check that everywhere we keep this consistency. And also probably it's better everything related to an AI give this suffix/preffix AI, which will represent us about it

Reply:
need both remark and description, please see comments above. Might have sense to add AI prefix